doc/oil-help.md

```diff
@@ -32,7 +32,7 @@ This section describes how to use the Oil binary.
     Usage: oil  [OPTION]... SCRIPT [ARG]...
            oil [OPTION]... -c COMMAND [ARG]...
 
-`bin/oil` is the same as `bin/osh` with a the `oil:all` option group set.  So
+`bin/oil` is the same as `bin/osh` with a the `oil:all` option groupset.  So
```

Comment:
The meaning I was going for was "option group enabled".

`oil:all` is an "option group", and the whole thing can be enabled or disabled (set or unset).